jolt-core/src/poly/one_hot_polynomial.rs

```diff
@@ -148,7 +148,7 @@ impl<F: JoltField> OneHotPolynomialProverOpening<F> {
             let univariate_poly_evals: [F; 2] = (0..B.len() / 2)
                 .into_par_iter()
                 .map(|k_prime| {
-                    let B_evals = B.sumcheck_evals_array::<2>(k_prime, BindingOrder::HighToLow);
+                    let B_evals = B.sumcheck_evals(k_prime, 2, BindingOrder::HighToLow);
```

Comment:
I was getting intermitted stack overflows during the batch opening proof sumcheck, so reverting these